cmd/get/matchable_execution_queue_attribute.go

```diff
@@ -0,0 +1,80 @@
+package get
+
+import (
+	"context"
+
+	"github.com/flyteorg/flytectl/cmd/config"
+	sconfig "github.com/flyteorg/flytectl/cmd/config/subcommand"
+	"github.com/flyteorg/flytectl/cmd/config/subcommand/executionqueueattribute"
+	cmdCore "github.com/flyteorg/flytectl/cmd/core"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
+)
+
+const (
+	executionQueueAttributesShort = "Gets matchable resources of execution queue attributes"
+	executionQueueAttributesLong  = `
+Retrieves execution queue attributes for given project and domain combination or additionally with workflow name.
+
+Retrieves execution queue attribute for project and domain
+Here the command get execution queue attributes for  project flytectldemo and development domain.
+::
+
+ flytectl get execution-queue-attribute -p flytectldemo -d development 
```

Comment:
can you add an example with a workflow name too?

Reply:
Done. Added for this and other attributes aswell.